flink-table/flink-sql-client/src/main/java/org/apache/flink/table/client/gateway/local/LocalExecutor.java

```diff
@@ -281,13 +283,15 @@ public void setSessionProperty(String sessionId, String key, String value) throw
 		ExecutionContext<?> context = getExecutionContext(sessionId);
 		Environment env = context.getEnvironment();
 		Environment newEnv = Environment.enrich(env, ImmutableMap.of(key, value), ImmutableMap.of());
+		ExecutionContext.SessionState sessionState = context.getSessionState();
```

Comment:
Do we need this change? It seems they are equivalent.

Reply:
I revert this